src/Views/AppInfoView.vala

```diff
@@ -310,7 +310,16 @@ public class AppCenter.Views.AppInfoView : AppCenter.AbstractAppContainer {
                 oars_flowbox.add (uncurated);
             }
 #endif
+
+#if HAS_APPSTREAM_1_0
+            var active_locale = "en_US";
+            var languages = package_component.get_languages ();
+            if (languages.length () > 0) {
+                active_locale = languages.nth_data (0);
+            }
```

Comment:
I think the `active_locale` here is essentially the user's language, not the first language of the package/component, so we can't always assume that the first language on the component is what we want to compare against.

However, I'm not sure how correct the existing code is, the `en_US` check feels weird to me, so does splitting the locale from the string.

But, these changes should be closer to the original AppStream < 1.0 code.

```suggestion
            var active_locale = "en_US";
            if (package_component.get_context () != null)
                active_locale = package_component.get_context ().get_locale () ?? "C";
            }
```

Reply:
Ah yeah so what this is intended to do is get the user's current locale and then immediately after this we check if it's US English and if it is we don't show localization information because US English will always be 100% and report 0%